docs/shared-services-cluster/08_5_minio_ssc.md

```diff
@@ -0,0 +1,46 @@
+# Install Minio
+
+We will deploy Minio as a target for harbor images and backups.
+
+This is a minimalist and POC quality deployment of Minio.  This is not a component of Tanzu.  This deployment is just for the purpose of demonstration purpose.
+
+## Set configuration parameters
+
+The scripts to prepare the YAML to deploy minio depend on a parameters to be set.  Ensure the following are set in `params.yaml`:
+
+```yaml
+minio:
+  server-fqdn: minio.dorn.tkg-aws-e2-lab.winterfell.live
+  root-user: foo
+  root-password: bar
+```
```

Comment:
Missed the `minio.persistence-size` property here

Reply:
Thanks, I updated the readme and added a default in the script.